# libloading 0.8.9: Kandelo's libc is musl; its RTLD_* values are Linux's.
--- a/src/os/unix/consts.rs	2026-10-08 15:41:34
+++ b/src/os/unix/consts.rs	2026-10-08 15:41:34
@@ -64,6 +64,7 @@
             pub(super) const RTLD_LAZY: c_int = 4;
         } else if #[cfg(any(
             target_os = "linux",
+            target_os = "kandelo",
             target_os = "android",
             target_os = "emscripten",
 
@@ -103,6 +104,7 @@
             pub(super) const RTLD_NOW: c_int = 1;
         } else if #[cfg(any(
             target_os = "linux",
+            target_os = "kandelo",
             all(target_os = "android", target_pointer_width = "64"),
             target_os = "emscripten",
 
@@ -165,6 +167,7 @@
             pub(super) const RTLD_GLOBAL: c_int = 8;
         } else if #[cfg(any(
             target_os = "linux",
+            target_os = "kandelo",
             all(target_os = "android", target_pointer_width = "64"),
             target_os = "emscripten",
 
@@ -209,6 +212,7 @@
             pub(super) const RTLD_LOCAL: c_int = 4;
         } else if #[cfg(any(
             target_os = "linux",
+            target_os = "kandelo",
             target_os = "android",
             target_os = "emscripten",
 
